tests/v2/validation/provisioning/resources/aws-out-of-tree.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: helm.cattle.io/v1
```

Comment:
I understand where these resources dir coming from, what do you think about calling this dir something like `outoftree/aws.yml` or `out-of-tree/aws.yml`? The resources dir in the charts are just a single step and their scope will always be a single step. But I feel like this one is more important than those resources so I think we should highlight the dir name somehow.

Namings I commented are placeholders, so totally your call on what to rename this `resources` dir.

Reply:
would `resources/out-of-tree/aws.yml` work?